primitives/src/vid/advz.rs

```diff
@@ -175,7 +197,7 @@ where
     evals: Vec<KzgEval<E>>,
 
     #[serde(with = "canonical")]
-    aggregate_proof: KzgProof<E>,
+    aggregate_proofs: Vec<KzgProof<E>>,
```

Comment:
```suggestion
    // aggretate_proofs.len() equals self.multiplicity
    // TODO further aggregate into a single KZG proof.
    aggregate_proofs: Vec<KzgProof<E>>,
```
...or some reasonable explainer comment.